var/www/so/capme/.inc/callback-elastic.php

```diff
@@ -278,17 +278,39 @@ function cliscript($cmd, $pwd) {
 				$errMsgElastic = "Second ES query didn't return a total number of hits.";
 			} elseif ( $elastic_response_object["hits"]["total"] == "0") {
 				$errMsgElastic = "Second ES query couldn't find this ID.";
-			/*} elseif ( $elastic_response_object["hits"]["total"] != "1") {
-				$errMsgElastic = "Second ES query returned multiple matches.";*/
 			} elseif ( ! isset($elastic_response_object["hits"]["hits"][0]["_source"]["protocol"]) ) {
 				$errMsgElastic = "Second ES query didn't return a protocol field.";
 			} elseif ( !in_array($elastic_response_object["hits"]["hits"][0]["_source"]["protocol"], array('tcp','udp'), TRUE)) {
                 		$errMsgElastic = "CapMe currently only supports TCP and UDP.";
```

Comment:
Since these last 2 `elseif` statements are referencing the first item in the array, they should be moved into the else block and updated to reference `$key`.

Reply:
Good call - will do.